src/System.IO.Compression.ZipFile/tests/ZipFileConvenienceMethods.cs

```diff
@@ -15,28 +17,29 @@ public class ZipFileTest_ConvenienceMethods : ZipFileTestBase
         [Fact]
         public async Task CreateFromDirectoryNormal()
         {
-            await TestCreateDirectory(zfolder("normal"), true);
-        }
+            string folderName = zfolder("normal");
+            string noBaseDir = GetTestFilePath();
+            ZipFile.CreateFromDirectory(folderName, noBaseDir);
 
-        [Fact]
-        [Trait(XunitConstants.Category, XunitConstants.IgnoreForCI)] // Jenkins fails with unicode characters [JENKINS-12610]
-        public async Task CreateFromDirectoryUnicodel()
-        {
-            await TestCreateDirectory(zfolder("unicode"), true);
+            await IsZipSameAsDirAsync(noBaseDir, folderName, ZipArchiveMode.Read, true, true);
```

Comment:
Done. I removed the unnecessary negative and updated it for Compression as well as Compression.ZipFile.